x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/alert_stats_collector.ts

```diff
@@ -0,0 +1,210 @@
+      };
+      alerts_closed: {
+        doc_count: number;
+      };
+    };
+    vuln_mgmt: {
+      rules_count: {
+        value: number;
+      };
+      alerts_open: {
+        doc_count: number;
+      };
+      alerts_acknowledged: {
+        doc_count: number;
+      };
+      alerts_closed: {
+        doc_count: number;
+      };
+    };
+  };
+}
+
+const getAlertsStatsQuery = (index: string) => ({
+  size: 0,
+  query: {
+    bool: {
+      filter: [{ term: { 'kibana.alert.rule.tags': 'Cloud Security' } }],
+    },
+  },
+  sort: '@timestamp:desc',
+  index,
+  aggs: {
+    cspm: {
+      filter: {
+        term: {
+          'kibana.alert.rule.tags': 'CSPM',
+        },
+      },
+      aggs: {
+        rules_count: {
+          cardinality: {
+            field: 'kibana.alert.rule.uuid',
+          },
+        },
+        alerts_open: {
+          filter: {
+            term: {
+              'kibana.alert.workflow_status': 'open',
+            },
+          },
+        },
+        alerts_acknowledged: {
+          filter: {
+            term: {
+              'kibana.alert.workflow_status': 'acknowledged',
+            },
+          },
+        },
+        alerts_closed: {
+          filter: {
+            term: {
+              'kibana.alert.workflow_status': 'closed',
+            },
+          },
+        },
+      },
+    },
+    kspm: {
+      filter: {
+        term: {
+          'kibana.alert.rule.tags': 'KSPM',
+        },
+      },
+      aggs: {
+        rules_count: {
+          cardinality: {
+            field: 'kibana.alert.rule.uuid',
+          },
+        },
+        alerts_open: {
+          filter: {
+            term: {
+              'kibana.alert.workflow_status': 'open',
+            },
+          },
+        },
+        alerts_acknowledged: {
+          filter: {
+            term: {
+              'kibana.alert.workflow_status': 'acknowledged',
+            },
+          },
+        },
+        alerts_closed: {
+          filter: {
+            term: {
+              'kibana.alert.workflow_status': 'closed',
+            },
+          },
+        },
+      },
+    },
+    vuln_mgmt: {
+      filter: {
+        term: {
+          'kibana.alert.rule.tags': 'CNVM',
+        },
+      },
+      aggs: {
+        rules_count: {
+          cardinality: {
+            field: 'kibana.alert.rule.uuid',
+          },
+        },
+        alerts_open: {
+          filter: {
+            term: {
+              'kibana.alert.workflow_status': 'open',
+            },
+          },
+        },
+        alerts_acknowledged: {
+          filter: {
+            term: {
+              'kibana.alert.workflow_status': 'acknowledged',
+            },
+          },
+        },
+        alerts_closed: {
+          filter: {
+            term: {
+              'kibana.alert.workflow_status': 'closed',
+            },
+          },
+        },
+      },
+    },
+  },
+});
+
+export const getAlertsStats = async (
```

Comment:
worth adding tests for that so it will cover future changes in the DE alerts implementation